mobile/ios/Runner/Info.plist

```diff
@@ -11,7 +11,7 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>$(DEVELOPMENT_LANGUAGE)</string>
 	<key>CFBundleDisplayName</key>
-	<string>AirQo</string>
```

Comment:
What inspired this change to replacing a variable with an actual value? @Mnoble-19

Reply:
And where is AirQo_dev as mentioned in the PR description? @Mnoble-19

Reply:
> What inspired this change to replacing a variable with an actual value? @Mnoble-19

Good morning @Baalmart , it's the other way round. This key holds the app name. Originally it was only one name for the app, already hard-coded. 
 As we're now using flavours, there is more than possible name  of the app i.e `AirQo` and `AirQo Dev`, thus I put variable and the name will be determined during the build

> And where is AirQo_dev as mentioned in the PR description? @Mnoble-19
I've added some screenshots to show the output

Reply:
@Baalmart I think you comments have been responded to.